tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -1030,6 +1087,7 @@ def rbd_mirror_config(**kw):
             imagesize=kw["config"]["rep_pool_config"]["size"],
             io_total=kw["config"]["rep_pool_config"]["io_total"],
             mode=kw["config"]["rep_pool_config"]["mode"],
+            mirrormode=kw["config"]["rep_pool_config"]["mirrormode"],
```

Comment:
i think we have to create mirrormode config , if we are passing in implicitly in initial_mirror_config mode method like we have created for the mode..
Are we expecting this config to pass from test suite file?
As you have hard-coded the config for this test suite it will work , i think it will fail for other test module.

Reply:
Thanks for the catch, good one
changed it.